CondTools/SiStrip/plugins/SiStripThresholdReader.cc

```diff
@@ -1,6 +1,33 @@
-#include "CondTools/SiStrip/plugins/SiStripThresholdReader.h"
+// user include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "CondFormats/SiStripObjects/interface/SiStripThreshold.h"
 #include "CondFormats/DataRecord/interface/SiStripThresholdRcd.h"
 
+// system include files
+#include <memory>
+#include <iostream>
+#include <cstdio>
+#include <sys/time.h>
+
+class SiStripThresholdReader : public edm::one::EDAnalyzer<> {
+public:
+  explicit SiStripThresholdReader(const edm::ParameterSet&);
+  ~SiStripThresholdReader() override = default;
+
+  void analyze(const edm::Event&, const edm::EventSetup&) override;
+
+private:
+  uint32_t printdebug_;
```

Comment:
shouldn't this be a bool after all?

Reply:
No, as you can see here it's used to limit the amount of printout.

https://github.com/cms-sw/cmssw/blob/8583b733fd4047cdd9fbeb285110bdae83c73c07/CondTools/SiStrip/plugins/SiStripThresholdReader.cc#L19

I am dismissing your comment.